cowprotocol/accounting/rewards/mainnet/vouch_registry_query_1541516.sql

```diff
@@ -0,0 +1,104 @@
+with
+
+last_block_before_timestamp as (
+    select max(number) from ethereum.blocks
+    where time < cast('{{end_time}}' as timestamp)
+),
```

Comment:
This could in principle call out to query 3333356. Then it would be easier to make sure that the convention on which boundary points are included is consistent across queries.

Reply:
Yeah thought about it. There is a start_time in query query 3333356, but i guess i could hardcode some very old one to work around it.